components/quest-boost/boostCard.tsx

```diff
@@ -60,6 +60,12 @@ const BoostCard: FunctionComponent<BoostCardProps> = ({
     [userBoostCheckStatus]
   );
 
+
+  const isBoostCompleteWithAddress = useMemo(
```

Comment:
why is this check not added in the the `isClickable` fn? since you use both of them together